sf_stats.Api/sf_stats.Domain/Entities/TeamSeason.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+
+namespace sf_stats.Domain.Entities
+{
+    public class TeamSeason : DbEntity
+    {
+        public int TeamId { get; set; }
+        public int SeasonId { get; set; }
+
+        public virtual Team Team { get; set; }
+        public virtual Season Season { get; set; }
+        public virtual List<Game> HomeGames { get; set; }
```

Comment:
This is kind of confusing. I am skeptical if this will work the way you think. I don't think you could distinguish between home and away games like this. 

Maybe we just add the team, season, and players for now and add the games later

Reply:
I'll relook at the ConfigurationBuilder. There (supposed to be) ways to have 2 columns point to the same table (TeamSeason) and have HomeGames populate with the entities that have the TeamSeasonId in the Home column, and have AwayGames populate with entities where the TeamSeasonId is in the away column. I can try and test this more to see. Probably not a bad idea to seed a full set of season data to make sure everything takes the way we expect.